go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/google/go-safeweb
 
-go 1.14
```

Comment:
Any reason for this?

Reply:
Well, you've deleted the whole file now. Keep the `go.mod` as it was because it's needed. Just don't change the Go version.

Reply:
Wups, sorry